packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -591,5 +623,17 @@ export async function makeWallet({
     getDepositFacetId,
   });
 
+  // Make Zoe invite purse
+  const inviteIssuerP = E(zoe).getInviteIssuer();
+  const addZoeIssuer = issuerP =>
```

Comment:
My preference would be to only use the `ZOE_*_PETNAME` constants in this code below, and not rely on them elsewhere.  Again, YMMV.

Reply:
That sounds like a good idea